source/instructors/authoring/assessments/ai-assessment-generation.rst

```diff
@@ -21,8 +21,37 @@ Important points to consider when auto-generating assessments:
 
 This is a new feature, and if it does not work properly for you, please let us know via the support chat or by emailing help@codio.com. If you would like us to take a look, send us your course name, the module, the assignment, and the page.
 
+.. note:: Not all assessments can be AI-generated. If you don't see a "Generate" button in the bottom right corner when you click on an assessment, that assessment type does not support AI generation. 
+
+
+How to Generate Assessment with AI
```

Comment:
Just a suggestion, it feels like it's missing the word "an"

How to Generate an Assessment with AI